demos/TicTacToe/Classes/MainMenuScene.cpp

```diff
@@ -0,0 +1,122 @@
+#include "MainMenuScene.h"
+
+#include "TicTacToeScene.h"
+
+USING_NS_CC;
+
+Scene* MainMenuScene::createScene() {
+  // Builds a simple scene that uses the bottom left cordinate point as (0,0)
+  // and can have sprites, labels and layers added onto it.
+  auto scene = Scene::create();
+  auto layer = MainMenuScene::create();
+
+  scene->addChild(layer);
+
+  return scene;
+}
+
+bool MainMenuScene::init() {
+  if (!Layer::init()) {
+    return false;
+  }
+  // Creates a sprite for the create button and sets its position to the center
+  // of the screen. TODO(grantpostma@): Dynamically choose the location.
+  auto create_button = Sprite::create("create_game.png");
+  create_button->setPosition(300, 350);
+  // Create a button listener to handle the touch event.
+  auto create_button_touch_listener = EventListenerTouchOneByOne::create();
+  // Setting the onTouchBegan event up to a lambda tha will replace the MainMenu
+  // scene with a TicTacToe scene.
+  create_button_touch_listener->onTouchBegan = [=](Touch* touch,
```

Comment:
I'm a little wary of [=] because you can potentially copy large objects unintentionally. Can you make your captures explicit?